assets/css/global.less

```diff
@@ -874,6 +874,246 @@ a.caret-right{
   }
 }
 
+.admonitionblock  {
+  border-top: 11px solid #06a6fd;
+  background-color: #f5f9fa;
```

Comment:
Should these colors be defined in variables? Might be easier to make a style guide from it later...